### PR TITLE
Fix latest eap

### PR DESCRIPTION
**Pull request by mallowigi**
Fix icon replacement on latest EAPs + Fix plugin load on Windows (thanks to @DevVersion - https://github.com/ChrisRM/material-theme-jetbrains/issues/245)

### CODE REVIEW

File: src/main/java/com/chrisrm/idea/utils/IconReplacer.java [hunk 1]
```diff
@@ -23,11 +26,16 @@ public static void replaceIcons(Class iconsClass, String iconsRootPath) {
                         setFieldValue(value, "myWasComputed", Boolean.FALSE);
                         setFieldValue(value, "myIcon", null);
                     } else if (byClass.getName().endsWith("$CachedImageIcon")) {
-                        patchUrlIfNeeded(value, iconsRootPath);
+                        String p = patchUrlIfNeeded(value, iconsRootPath);
+                        Icon newIcon = IconLoader.getIcon(p);
+						setFinalStatic(field, newIcon);
```
Comment (Mheetu) [inline src/main/java/com/chrisrm/idea/utils/IconReplacer.java]: A bit nit-picky, but I'd fix the indentation here and give the variable `p` a proper name.

**Comment by gayanper** [comment]
@ChrisRM  When can we expect this to be merged into master ?

**Comment by Mheetu** [comment]
My only concern is rather or not this would be backwards compatible since not everybody will be using the latest EAP... Aside from that, I'd like to see it merged.

**Comment by gayanper** [comment]
> My only concern is rather or not this would be backwards compatible since not everybody will be using the latest EAP... Aside from that, I'd like to see it merged.

I don't think it will cause i found a issue while trying to compile the changes in pull branch in MTLaf class where the log method needs to be made public to shadow the method in DarculaLaf.

So should we branch off :) while keeping a release branch for pre 16.2.x versions ?

**Comment by mallowigi** [comment]
Why did we need windows icons anyway? Where were they used for ?

**Comment by gayanper** [comment]
Another thing i notice is in this pull request branch is that the breadcrumbs color for xml documents doesn't use the correct colors as you can see below
image

**Comment by mallowigi** [comment]
@gayanper breadcrumbs color are in a new section in the latest EAP, under General > Editor > Breadcrumbs

**Comment by mallowigi** [comment]
bump

**Comment by SalahAdDin** [comment]
:+1:

**Comment by christianpohl** [comment]
👍

**Comment by gayanper** [comment]
The latest version of this pull request doesn't show the correct icons for methods.

**Pull request closed without merging**